src/props.js

```diff
@@ -18,7 +18,11 @@ var mapToEntries = (function() {
         index++;
     }
 
-    return function mapToEntries(map) {
+    // NOTE: this used to be named function
+    // but making it anonymous to make safari
+    // happy. safari did not like
+    // optimized/mangled output of this named function.
```

Comment:
What was the symptom? Can you mention the actual bug here (e.g. app did not launch in Safari).